src/greyVest/Grid.js

```diff
@@ -16,9 +16,11 @@ let Grid = ({
   placeItems,
   inline = false,
   style,
+  className,
   ...props
 }) => (
   <Component
+    className={F.compactJoin(['gv-grid', className])}
```

Comment:
compact join takes the join string as it first param - I think you mean 

```js
F.compactJoin(' ', ['gv-grid', className])
```

Reply:
Removed the easter egg :)